spec/controllers/admin/companies_controller_spec.rb

```diff
@@ -26,7 +26,7 @@
 
     it "redirects back to admin dashboard index after deletion" do
```

Comment:
I would change the name of this test... to be redirects to admin companies index instead of admin dashboard, just to be clear...